File: starting_filesets/C#/Prime Factors/PrimeFactorsTest.cs

using NUnit.Framework;

[TestFixture]
public class PrimeFactorsTest
{
    [Test]
    public void Example()
    {
        int expected = 6 * 9;
        int actual = new PrimeFactors().Answer;
        Assert.AreEqual(expected, actual);
    }
}

File: starting_filesets/Java/Prime Factors/PrimeFactorsTest.java

import org.junit.*;
import static org.junit.Assert.*;

public class PrimeFactorsTest 
{
    @Test
    public void example()
    {
        int expected = 6 * 9;
        int actual = new PrimeFactors.answer();
        assertEquals(expected, actual);
    }
}
